Demo/Sources/Examples/ExamplesView.swift

```diff
@@ -80,15 +80,31 @@ struct ExamplesView: View {
     @EnvironmentObject private var router: Router
 
     var body: some View {
+#if os(iOS)
         List {
+            content()
+        }
+        .navigationTitle("Examples")
+#else
+        ScrollView {
+            VStack(alignment: .leading) {
+                content()
+                    .padding(.horizontal, 50)
+            }
+        }
+#endif
+    }
+
+    @ViewBuilder
+    private func content() -> some View {
+        Group {
             MediaEntryView()
             srgSections()
             thirdPartySections()
             miscellaneousSections()
         }
         .scrollDismissesKeyboard(.immediately)
         .animation(.defaultLinear, value: model.protectedMedias)
-        .navigationTitle("Examples")
         .tracked(name: "examples")
         .refreshable { await model.refresh() }
```

Comment:
In `ContentListView`, `refreshable` is applied on iOS only. Is it applied to every platform on purpose here?

Reply:
It's an oversight on my part, thanks!